### ISSUE TITLE
Set outline on a component-by-component basis

### ISSUE THREAD

**Issue by arussell123** [molstar/molstar#634]
Hi, 

I am trying to make polymers less visible so they are not too intrusive, we have reduced the transparency of the component however with outlines on they are still very visible. Is it possible to configure the outline on a component-by-component basis?

In previous molstar versions it seemed to be the case that the outlines were not shown if the component hit a certain transparency but it looks like this has changed. If we can't do it on a component basis is it possible to set a transparency threshold so that when a component hits it it does not show the outline.

Thanks!

**Comment by arose** [comment]
The outline is a post-processing effect that can't be done per component. It would be possible to switch it off via a define in https://github.com/molstar/molstar/blob/master/src/mol-gl/shader/outlines.frag.ts. Care to do a PR?

**Comment by arussell123** [comment]
It is already possible to turn off the outline for everything. In my case we want the outline for most things but not the components which we've made transparent (by reducing the alpha value). I've got it working locally for me to have an additional input param to set an minimum alpha value and it only adds the outline for the components with a higher alpha value - I'll put up a PR for my solution and link it here, but let me know if you have any suggestions.
